xarray/tests/test_dataset.py

```diff
@@ -6191,6 +6191,33 @@ def test_raise_no_warning_for_nan_in_binary_ops():
     assert len(record) == 0
 
 
+@pytest.mark.parametrize("ds", (2,), indirect=True)
+def test_raise_no_warning_assert_close(ds):
+
+    with pytest.warns(None) as record:
+        assert_allclose(ds, ds)
+
+    assert len(record) == 0
+
+
+@pytest.mark.xfail()
+@pytest.mark.parametrize("ds", (2,), indirect=True)
+@pytest.mark.parametrize("name", ("mean", "max"))
+def test_raise_no_warning_dask_rolling_assert_close(ds, name):
```

Comment:
This is a puzzle — I can't easily find the source of the warning. It requires `assert_allclose` to be run, for the `ds` param to be 2, and is different for `mean` and `max`. `sum` raises no warning.

Unless we can easily figure this out, I'd be up for xfailing, leaving a note, and merging this — it does seem to be a corner case.

Reply:
can you add a reason to the xfail? That way understanding the test might be a little easier.